src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -144,9 +144,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NuGet.Core" Version="2.12.0" />
-    <PackageReference Include="NuGet.VisualStudio" Version="3.3.0" />
```

Comment:
We no longer use any of the types in these packages so we can drop the references to them.